app/src/androidTest/java/xyz/jhughes/laundry/Matchers/ViewChild.java

```diff
@@ -0,0 +1,53 @@
+package xyz.jhughes.laundry.Matchers;
+
+import android.view.View;
+import android.view.ViewGroup;
+
+import org.hamcrest.Description;
+import org.hamcrest.Matcher;
+import org.hamcrest.TypeSafeMatcher;
+
+/**
+ * Created by mvieck on 4/24/17.
+ */
+
+public class ViewChild {
+    public static Matcher<View> firstChildOf(final Matcher<View> parentMatcher) {
+        return new TypeSafeMatcher<View>() {
+            @Override
+            public void describeTo(Description description) {
```

Comment:
Does this mean we can print out descriptions of our tests? Do other tests have a way for this or is this just for when it fails?